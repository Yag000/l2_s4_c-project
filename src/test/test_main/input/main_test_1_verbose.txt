debug
debug one
debug one two
debug                             one two
    debug one two  
    debug     one   two  
debug one two three