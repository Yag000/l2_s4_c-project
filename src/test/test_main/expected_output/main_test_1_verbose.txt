Parsing file src/test/test_main/input/main_test_1_verbose.txt ...
/$ echo
/$ echo one
one
/$ echo one two
one two
/$ echo one two
one two
/$ echo one two
one two
/$ echo one two
one two
/$ echo one two
one two
